src/directives/ec-money/utils.spec.js

```diff
@@ -0,0 +1,70 @@
+import { format, unFormat, sanitizeInput } from './utils';
```

Comment:
We should check misplaced group separators e.g. `1,1,1,1,` or `111,,111`
and also check multiple decimal separators e.g. `1..11` or `1.1.11`